R/tlf_ae_summary.R

```diff
@@ -43,7 +43,7 @@ tlf_ae_summary <- function(outdata,
                            col_rel_width = NULL,
                            text_font_size = 9,
                            orientation = "portrait",
-                           title = NULL,
+                           title = c("analysis", "observation", "population"),
```

Comment:
Is there a reason why we need to give 3 options to trigger this? 
What happens if someone passes in a character vector of 
`c("analysis", "this is a real title", "this is another real title")` and expect those to literally be the titles of their output?

Reply:
Changing the default value may be related to my recent PR to extend the flexibility of defining title https://github.com/Merck/metalite/blob/main/R/collect.R#L286

We probably should change the argument to `title_order` to be aligned with `metalite`.

There should be a input checking to only allow component name to be used. 

The actual title should be defined while user define a `metalite` object and reused in different places.